Cargo.toml

```
@@ -178,6 +177,7 @@ similar-asserts = "1.4.2"
 slotmap = { version = "1.0.6", features = ["serde"] }
 smallvec = { version = "1.0", features = ["const_generics", "union"] }
 static_assertions = "1.1"
+# update of strum blocked by arrow2/comfytable
 strum = { version = "0.24", features = ["derive"] }
```

To make future sorting easier
```suggestion
strum = { version = "0.24", features = ["derive"] } # update of strum blocked by arrow2/comfytable

---

I wish, but the  `.toml` auto-format then reflows that entire line, spreading it out over three lines, making it much harder to sort (yes, this was my first instinct too)